components/Wishlist/wishlist.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>E-commerce | products</title>
+		<script
+			src="https://kit.fontawesome.com/6056b5574b.js"
+			crossorigin="anonymous"
+		></script>
+		<link
+			rel="stylesheet"
+			href="https://xero-cl-netlify.netlify.app/css/component.css"
+		/>
+		<link rel="stylesheet" href="/css/component.css" />
+	</head>
+	<body>
+		<div class="component-display-container nav-bar">
+			<div class="nav-container">
+				<div class="nav-brand"
+					><img class="logo" src="/assets/logo.svg" alt="logo" />
+					<a class="component-library-link nav-icons" href="../../index.html">
+						<i class="fa">Xero</i></a
+					></div
+				>
+				<div class="search-bar">
+					<input
+						class="input"
+						placeholder="Search for products, brands and more..."
```

Comment:
the placeholder is not totally visible, either increase the width or shorten the text of the placeholder.
![placeholder](https://user-images.githubusercontent.com/95222845/154899609-6ecb2313-1b10-4ea4-aa52-f47946542823.png)